Cargo.toml

```diff
@@ -199,6 +199,9 @@ serialize = ["bevy_internal/serialize"]
 # Enables multithreaded parallelism in the engine. Disabling it forces all engine tasks to run on a single thread.
 multi-threaded = ["bevy_internal/multi-threaded"]
 
+# Use async-io's implementation of block_on instead of futures-lite's implementation. This is preferred if your application uses async-io.
+async-io = ["bevy_internal/async-io"]
```

Comment:
I don't understand why this is showing up in the diff. It is merged into main: https://github.com/bevyengine/bevy/blob/main/Cargo.toml#L202 @cart this is the odd thing that I saw the other week where github's PR diff seems to get confused.